plugins/pluginmanager.go

```diff
@@ -52,17 +83,18 @@ func PluginManagerConfigSchema() schematypes.Object {
 		MetaData: schematypes.MetaData{
 			Title: "Plugin Configuration",
 			Description: `Mapping from plugin name to plugin configuration.
-                    The 'disabled' key is special and lists plugins that are
-                    disabled. Plugins that are disabled do not require
-                    configuration.`,
+										A plugin is enabled if it has an entry in this mapping, and
+										isn't explicitly listed as 'disabled'. Even plugins that
+										don't require configuration must have an entry, in these
+										cases, empty object will suffice.`,
```

Comment:
Yeah, we should use tabs for indentation inside multi-line strings...
I'm working on a tiny`util.Markdown` hack that will remove the indentation and replace `'` with ` so that we can easily write docs...